package.json

```diff
@@ -6,26 +6,33 @@
     "node": "10.16.0"
   },
   "devDependencies": {
+    "babel-core": "^6.26.3",
```

Comment:
May we use the latest versions of these 4 from babel? https://babeljs.io/docs/en/v7-migration